app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    if command[0] == "cp":
```

Comment:
check if command has 3 elements before it

Reply:
also you can use unpacking 
`command_name, file_name, copy_file_name = command.split()`